content_sync/pipelines/definitions/concourse/common/resource_types.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, **kwargs):
         super().__init__(
             name=HTTP_RESOURCE_TYPE_IDENTIFIER,
             type=REGISTRY_IMAGE,
-            source={"repository": "jgriff/http-resource", "tag": "latest"},
+            source={"repository": "umar8hassan/http-resource", "tag": "latest"},
```

Comment:
Upon testing this, I was getting the following error on both the `ocw-studio` webhook step as well as the offline build gate:

`exec /opt/resource/out: exec format error`

This is because you have complied this only for `arm64`:

https://hub.docker.com/r/umar8hassan/http-resource/tags

![image](https://github.com/user-attachments/assets/0e8c2c83-1840-4f20-8b13-17671684bbcf)

This needs to be compiled for x86.